backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/IndexCreationFactory.java

```diff
@@ -115,6 +115,8 @@ private void createIndexIfNeeded(RestHighLevelClient client, IndexName indexName
             } catch (ElasticsearchStatusException exception) {
                 if (exception.getMessage().contains(INDEX_ALREADY_EXISTS_EXCEPTION_MESSAGE)) {
                     LOGGER.info("Index [{}] already exist", indexName);
+                } else if (exception.getMessage().contains(INDEX_NAME_EXCEPTION_MESSAGE)) {
+                    LOGGER.info("Index [{}] already exist as alias", indexName);
```

Comment:
When can it happen?

Reply:
TBH I'm surprised it happens at all. But the test I added is reproducing exactly the exception message from ES that we were gettting from time to time a week ago. Strangely enough though, it seems not happening anymore lately. I will fix the PR but we might wait to see if that error strikes again before thinking more on this. Agreed?